test/unit/events/mouse.js

```diff
@@ -250,4 +250,238 @@ suite('Mouse Events', function() {
+      sketches.end(); //resolve all sketches by calling their finish functions
+      let counts = await sketches.result; //get array holding number of times mouseReleased was called. Rejected sketches also thrown here
+      assert.deepEqual(counts, [1, 1]);
+    });
+  });
+
+  suite('mouseClicked', function() {
+    test('mouseClicked function must run when mouse is clicked', async function() {
+      let count = 0;
+
+      myp5.mouseClicked = function() {
+        count += 1;
+      };
+
+      window.dispatchEvent(new MouseEvent('click'));
+      assert.deepEqual(count, 1);
+    });
+
+    test('mouseClicked functions on multiple instances must run once', async function() {
+      let sketchFn = function(sketch, resolve, reject) {
+        let count = 0;
+
+        sketch.mouseClicked = function() {
+          count += 1;
+        };
+
+        sketch.finish = function() {
+          resolve(count);
+        };
+      };
+      let sketches = parallelSketches([sketchFn, sketchFn]); //create two sketches
+      await sketches.setup; //wait for all sketches to setup
+      window.dispatchEvent(new MouseEvent('click'));
+      sketches.end(); //resolve all sketches by calling their finish functions
+      let counts = await sketches.result; //get array holding number of times mouseClicked was called. Rejected sketches also thrown here
+      assert.deepEqual(counts, [1, 1]);
+    });
+  });
+
+  suite('doubleClicked', function() {
+    test('doubleClicked function must run when mouse is double clicked', async function() {
+      let count = 0;
+
+      myp5.doubleClicked = function() {
+        count += 1;
+      };
+
+      window.dispatchEvent(new MouseEvent('dblclick'));
+      assert.deepEqual(count, 1);
+    });
+
+    test('doubleClicked functions on multiple instances must run once', async function() {
+      let sketchFn = function(sketch, resolve, reject) {
+        let count = 0;
+
+        sketch.doubleClicked = function() {
+          count += 1;
+        };
+
+        sketch.finish = function() {
+          resolve(count);
+        };
+      };
+      let sketches = parallelSketches([sketchFn, sketchFn]); //create two sketches
+      await sketches.setup; //wait for all sketches to setup
+      window.dispatchEvent(new MouseEvent('dblclick'));
+      sketches.end(); //resolve all sketches by calling their finish functions
+      let counts = await sketches.result; //get array holding number of times doubleClicked was called. Rejected sketches also thrown here
+      assert.deepEqual(counts, [1, 1]);
+    });
+  });
+
+  suite('mouseWheel', function() {
+    test('mouseWheel function must run when mouse wheel event is detected', async function() {
+      let count = 0;
+
+      myp5.mouseWheel = function() {
+        count += 1;
+      };
+
+      window.dispatchEvent(new MouseEvent('wheel'));
+      assert.deepEqual(count, 1);
+    });
+
+    test('doubleClicked functions on multiple instances must run once', async function() {
```

Comment:
it looks like this test case is labelled incorrectly? i see the same description higher in the file and internally this uses `mouseWheel`